src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -88,7 +89,8 @@ private void MakeCompilationOutputRunnableForCoreCLR()
         private void CopyContentFiles()
         {
             var contentFiles = new ContentFiles(_context);
-            contentFiles.StructuredCopyTo(_runtimeOutputPath);
+            var includeEntries = _compilerOptions.CopyToOutputInclude?.GetIncludeFiles(PathUtility.EnsureTrailingSlash(_runtimeOutputPath));
```

Comment:
split out PathUtility.EnsureTrailingSlash(_runtimeOutputPath), this line is too long